pkg/crc/constants/constants.go

```diff
@@ -155,3 +153,17 @@ func GetCRCMacTrayDownloadURL() string {
 func GetCRCWindowsTrayDownloadURL() string {
 	return fmt.Sprintf(CRCWindowsTrayDownloadURL, version.GetCRCWindowsTrayVersion())
 }
+
+func GetDefaultCPUs(openshift bool) int {
+	if openshift {
+		return 4
+	}
+	return 2
+}
+
+func GetDefaultMemory(openshift bool) int {
+	if openshift {
+		return 9216
+	}
+	return 2048
+}
```

Comment:
This change also feels like it should be done the opposite way, first instantiate a `PresetDefaults`, and then call `presetDefaults.GetMemory()` and such?